src/test/perl/filewriter-notmocked.t

```diff
@@ -80,6 +80,15 @@ is(readfile($fn), $TEXT, "file $fn created with expected contents");
 is(get_perm($fn), 0764, "created file $fn has mode 0764");
 is((stat($fn))[9], 1234567, "mtime set to 1234567 forfile $fn");
 
+$fh = CAF::FileWriter->open ($fn, mode => 0764, log => $obj);
+print $fh $TEXT;
+ok(!$fh->close(), 'close reports no change w same content and same permissions');
+
+$fh = CAF::FileWriter->open ($fn, mode => 0700, log => $obj);
+print $fh $TEXT;
+ok($fh->close(), 'close reports change w same content and different permissions');
```

Comment:
I don't think there's any reason to keep lines short is there? In which case it would be better to have the full "with" instead of just w which is a bit of an idiom and might be harder for non-experts in English to comprehend?